BrainPortal/app/models/cbrain_task.rb

```diff
@@ -379,7 +379,14 @@ def short_description
     header
   end
 
+  ##########################################
+  # Access control (overrides module ResourceAccess)
+  ##########################################
 
+  def self.find_all_accessible_by_user(user, options = {}) #:nodoc:
+    scope = super
+    scope.where(:bourreau_id => Bourreau.find_all_accessible_by_user(user).pluck(:id))
```

Comment:
Should remove the restriction on the bourreau ?

Reply:
Yes, I'll return this to the standard CBRAIN code.